frontend/index.html

```diff
@@ -21,6 +21,7 @@
   </script>
 </head>
 <body class="theme-green">
+<a class="github-fork-ribbon" href="https://github.com/chromium/hstspreload.org" data-ribbon="On GitHub" title="On GitHub">On GitHub</a>
```

Comment:
For my own future reference: looks like getting a text attribute from CSS is well-supported:
https://developer.mozilla.org/en-US/docs/Web/CSS/attr